ci-operator/config/openshift/release/openshift-release-master__ci-4.9.yaml

```diff
@@ -201,13 +225,27 @@ tests:
     env:
       TEST_SUITE: openshift/conformance/parallel
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-techpreview-parallel
+  cron: 0 2,10,18 * * *
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      TEST_SUITE: openshift/conformance/parallel
+    workflow: openshift-e2e-openstack-techpreview
 - as: e2e-openstack-serial
   cron: 0 6,14,22 * * *
   steps:
     cluster_profile: openstack-vexxhost
     env:
       TEST_SUITE: openshift/conformance/serial
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-techpreview-serial
```

Comment:
Ohh, so we just have periodics for serial suite?

Reply:
We have serial and the regular tests (parallel?), the block above this should be for the openstack regular tests